test/services/lduService.test.js

```diff
@@ -59,8 +59,76 @@ describe('lduService', () => {
         description: 'London',
         ldus: [
           { code: 'ham', description: 'Hampstead', active: true },
+          { code: 'pic', description: 'Picadilly', active: true },
           { code: 'wtl', description: 'Waterloo', active: false },
+        ],
+      })
+    })
+
+    it('Should remove duplicate ldus and only show the first description', async () => {
+      deliusClient.getAllLdusForProbationArea.mockResolvedValue({
+        content: [
+          { code: 'ham', description: 'Hampstead' },
+          { code: 'pic', description: 'Picadilly' },
+          { code: 'ham', description: 'Hampstead2' },
+          { code: 'pic', description: 'Picadilly2' },
+          { code: 'wtl', description: 'Waterloo2' },
+          { code: 'pic', description: 'Picadilly' },
+          { code: 'ham', description: 'Hampstead2' },
+          { code: 'pic', description: 'Picadilly2' },
+          { code: 'wtl', description: 'Waterloo' },
+        ],
+      })
+
+      deliusClient.getAllProbationAreas.mockResolvedValue({ content: [{ code: 'Lon', description: 'London' }] })
+      activeLduClient.allActiveLdusInArea.mockResolvedValue([{ code: 'ham' }, { code: 'pic' }])
+
+      const result = await lduService.getProbationArea('Lon')
+
+      expect(result).toEqual({
+        code: 'Lon',
+        description: 'London',
+        ldus: [
+          { code: 'ham', description: 'Hampstead', active: true },
           { code: 'pic', description: 'Picadilly', active: true },
+          { code: 'wtl', description: 'Waterloo', active: false },
+        ],
+      })
+    })
+
+    it('Should NOT return duplicate ldus', async () => {
+      deliusClient.getAllLdusForProbationArea.mockResolvedValue({
+        content: [
+          { code: 'ham', description: 'Hampstead' },
+          { code: 'wtl', description: 'Waterloo' },
+          { code: 'pic', description: 'Picadilly' },
+          { code: 'ham', description: 'Hampstead2' },
+          { code: 'pic', description: 'Picadilly2' },
+          { code: 'wtl', description: 'Waterloo2' },
+          { code: 'ham', description: 'Hampstead' },
+          { code: 'wtl', description: 'Waterloo' },
+          { code: 'pic', description: 'Picadilly' },
+          { code: 'ham', description: 'Hampstead2' },
+          { code: 'pic', description: 'Picadilly2' },
+          { code: 'wtl', description: 'Waterloo2' },
+        ],
+      })
+
+      deliusClient.getAllProbationAreas.mockResolvedValue({ content: [{ code: 'Lon', description: 'London' }] })
+      activeLduClient.allActiveLdusInArea.mockResolvedValue([{ code: 'ham' }, { code: 'pic' }])
+
+      const result = await lduService.getProbationArea('Lon')
+
+      expect(result).not.toEqual({
```

Comment:
This isn't a very persuasive test. There's literally an infinite amount of things that results is not equal to. It's not equal to: `"some random string"`, `1`, `true`, a literal fish, the Milk Way, the concept of silence... etc..

Instead it would be more persuasive to show what it is equal to and then the reader can see that there are no duplicates in there, for themselves.

Reply:
which it looks like you already have - so this test is unnecessary?

Reply:
I did this test  to show that the positive test wasn't passing for all cases.
Now removed